e2e/Utils.ts

```diff
@@ -260,3 +261,21 @@ export const normalizeIcons = (str: string) => {
     .replace(new RegExp('\u00D7', 'g'), '\u2715')
     .replace(new RegExp('\u221A', 'g'), '\u2713');
 };
+
+// Certain environments (like CITGM and GH Actions) do not come with mercurial installed
+let hgIsInstalled: boolean | null = null;
+
+export const testIfHg = (
+  ...args: Parameters<typeof test>
+): ReturnType<typeof test> => {
+  if (hgIsInstalled === null) {
+    hgIsInstalled = which.sync('hg', {nothrow: true}) !== null;
+  }
+
+  if (hgIsInstalled) {
+    return test(...args);
+  } else {
+    console.warn('Mercurial (hg) is not installed - skipping some tests');
+    return test.skip(...args);
+  }
+};
```

Comment:
```suggestion
export const testIfHg = (
  ...args: Parameters<typeof test>
) => {
  if (hgIsInstalled === null) {
    hgIsInstalled = which.sync('hg', {nothrow: true}) !== null;
  }

  if (hgIsInstalled) {
    test(...args);
  } else {
    console.warn('Mercurial (hg) is not installed - skipping some tests');
    test.skip(...args);
  }
};
```

Reply:
I took your suggestions, I'll run prettier locally just in case

Reply:
Prettier should be fixed